package-lock.json

```diff
@@ -1,13 +1,53 @@
 {
```

Comment:
If you are not introducing new packages or updating package versions, please revert the changes in `package-lock.json`. `npm install` may mutate this file and creates a lot of unnecessary changes.